### DISCUSSION TITLE
Transfer Misskey account to or from Mastodon

### DISCUSSION THREAD

**Discussion by wardhale** [misskey-dev/misskey#9353] [Q&A] [answered]
I set up an instance recently, but before I make it live I want to make sure users can transfer their accounts to Misskey from Mastodon... or from Mastodon to Misskey. I can't seem to figure this out. In mastodon you have to make an alias of the old account you want to transfer from the new server/account... then go to your old account/server and move it to the new one.  Can Misskey do aliases? Can we move our accounts to mastodon instances or vice/versa?

**Comment by ThatOneCalculator** [comment]
You can't do that on stock Misskey, but you can on [Calckey](https://codeberg.org/calckey/calckey) :)

**Reply by wardhale** [reply]
Guess I'll have to head over the calckey! cheers.

**Comment by tamaina** [comment]
Account migration and moving is not yet implemented 

https://github.com/misskey-dev/misskey/issues/5475

**Reply by wardhale** [reply]
Cheers. Hopefully it'll be a thing in the future. I'll use calckey for now.

**Reply by ThisIsMissEm** [reply, marked as answer]
Update: it's implemented: https://github.com/misskey-dev/misskey/issues/5475#issuecomment-2810639262
